src/components/Hero/Hero.test.tsx

```diff
@@ -672,3 +673,114 @@ describe("Hero", () => {
     expect(container.querySelector("div")).toBe(ref.current);
   });
 });
+describe("Test getTextColor function (hero.ts)", () => {
+  let type, mode, foregroundColor, isDarkText;
+  it("returns foregroundColor", () => {
+    type = "heading";
+    mode = "light";
+    foregroundColor = "brand.primary";
+    isDarkText = false;
+    expect(getTextColor(type, mode, foregroundColor, isDarkText)).toBe(
+      "brand.primary"
+    );
+    type = "body";
+    expect(getTextColor(type, mode, foregroundColor, isDarkText)).toBe(
+      "brand.primary"
+    );
+  });
+  it("returns default heading text color", () => {
+    type = "heading";
+    mode = "light";
+    foregroundColor = undefined;
+    isDarkText = false;
+    expect(getTextColor(type, mode, foregroundColor, isDarkText)).toBe(
+      "ui.typography.inverse.heading"
```

Comment:
why does the default have `inverse`? I would expect it to just be `ui.typography.heading`.

Reply:
Yeah, it starts to get a little convoluted, but in the end it makes sense.  Our standard text color is dark (for a light background) and the inverse text color is light (for a dark background).  The default background colors for the Hero variants are dark, so the colors of the text that sits atop those dark background colors need to be light and that is the "inverse" of our standard text colors.